docs/libref.rst

```diff
@@ -16,8 +16,6 @@ Library Reference
 .. autoclass:: TransactionsEndpoint
     :members:
 
-    .. automethod:: __init__
-
```

Comment:
I don't think there's any point in including the `NamespacedDriver.__init__` in here.